tee-worker/scripts/litentry/release/release.sh

```diff
@@ -0,0 +1,336 @@
+      echo "Removing old log life"
+      rm -f $LOG
+      systemctl --user daemon-reload
+      systemctl --user start worker.service
+      print_divider
+      echo "Check logs for the upgraded worker here at /log/worker0/log"
+  else
+      export RUST_LOG='info,integritee_service=debug,ws=warn,sp_io=error,substrate_api_client=warn,
+      itc_parentchain_light_client=info,
+      jsonrpsee_ws_client=warn,jsonrpsee_ws_server=warn,enclave_runtime=debug,ita_stf=debug,
+      its_rpc_handler=warn,itc_rpc_client=warn,its_consensus_common=debug,its_state=warn,
+      its_consensus_aura=warn,aura*=warn,its_consensus_slots=warn,
+      itp_attestation_handler=debug,http_req=debug,lc_mock_server=warn,itc_rest_client=debug,
+      lc_credentials=debug,lc_identity_verification=debug,lc_stf_task_receiver=debug,lc_stf_task_sender=debug,
+      lc_data_providers=debug,itp_top_pool=debug,itc_parentchain_indirect_calls_executor=debug'
+
+      echo "Starting new worker"
+      cd tmp/w0
+
+      # Redirect stdout to a log file
+      log_file="../../log/worker0.log"
+
+      exec ./integritee-service -P 2000 -w 2001 -r 3443 -h 4545 --running-mode mock --enable-mock-server --parentchain-start-block 0 run --skip-ra --dev >"$log_file" 2>&1
+  fi
+}
+
+setup_working_dir() {
+    source_dir=$1
+    target_dir=$2
+
+    optional=("key.txt" "spid.txt")
+
+    for file in "${optional[@]}"; do
+        source="${source_dir}/${file}"
+        target="${target_dir}/${file}"
+
+        if [ -f "$source" ]; then
+            cp "$source" "$target"
+        else
+            echo "$source does not exist, this is fine, but you can't perform remote attestation with this."
+        fi
+    done
+
+    mandatory=("enclave.signed.so" "integritee-service")
+
+    for file in "${mandatory[@]}"; do
+        source="${source_dir}/${file}"
+        target="${target_dir}/${file}"
+
+        if [ -f "$source" ]; then
+            cp "$source" "$target"
+        else
+            echo "$source does not exist. Did you run make?"
+        fi
+    done
+}
+
+if [ "$UPGRADE_WORKER" = "1" ]; then
+  # Assuming the worker has already stopped
+  echo "Preparing to Upgrade Worker"
+  upgrade_worker
+  exit 0
+fi
+
+# Example usage
+# generate_service_file "my_service" "My Service Description" "/path/to/my_script.sh" "/path/to/working_directory"
+if [ "$PRODUCTION" = "1" ]; then
+    echo "Running in production mode."
+    setup_working_dir "bin" "tmp/w0"
+    generate_worker_service_file
+    generate_parachain_service_file
+else
+    echo "Not running in production mode."
+fi
+
+if [ "$AUTO_START" = "true" ]; then
+  # Start the services
+  if [ "$PRODUCTION" = "1" ]; then
+    echo "Starting Services"
+    start_service_files
+  else
+    ./local-setup/launch.sh
+  fi
+fi
+
+# TODO: Need to also perform clean reset of the service
+if [ "$CLEAN_RESET" = "true" ]; then
+  # Assuming the worker has already stopped
+  echo "Preparing to Upgrade Worker"
+fi
```

Comment:
This is a bit confusing - `clean_reset` normally happens in **resetting** the worker, not upgrading

Reply:
No longer needed, Using a new script `deploy.sh`